test/framework/config.go

```diff
@@ -145,9 +135,49 @@ func ReadConfig(fileName string, hl HostsList) (*TestsuiteConfig, error) {
 		}
 	}
 
+	// Merge variables from command line into default variable values
+	// from JSON configuration
+	for kkk, vvv := range vl {
+		config.Variables[kkk] = vvv
+	}
+
+	// Replace variables in test command lines
+	for ttt := range config.Tests {
+		for aaa := range config.Tests[ttt].Apps {
```

Comment:
Tripple letter variable names are unusual. `>.<`

Reply:
Variables like this are easier to search/replace in text editor.